community/CONTRIBUTING.md

```diff
@@ -65,45 +76,69 @@ Thanks for considering contributing to OPEA project. The contribution process is
+    |   |   │   ├── docker_compose
+    |   |   │   │   └── compose.yaml
+    |   |   │   └── kubernetes
+    |   |   │       ├── README.md
+    |   |   │       └── cpu-values.yaml
+    |   |   └── src
+    |   |       ├── Dockerfile
+    |   |       ├── README.md    # could be multiple files for different integrations
+    |   |       ├── __init__.py
+    |   |       ├── integrations
+    |   |       │   ├── __init__.py
+    |   |       │   ├── multimodal_bridgetower.py
+    |   |       │   ├── predictionguard.py
+    |   |       │   └── tei.py
+    |   |       ├── opea_embedding_microservice.py
+    |   |       ├── opea_multimodal_embedding_microservice.py
+    |   |       └── requirements.txt
+    |   ├── third_parties
+    |   │   ├── bridgetower
+    |   │   ├── clip
+    |   │   ├── elasticsearch
+    |   │   ├── gpt-sovits
+    |   │   ├── milvus
+    |   │   ├── mongodb
+    |   │   ├── nginx
+    |   │   ├── opensearch
+    |   │   ├── pathway
+    |   │   ├── pgvector
+    |   │   ├── redis
+    |   │   ├── speecht5
+    |   │   ├── tei
+    |   │   ├── teirerank
+    |   │   ├── tgi
+    |   │   ├── vdms
+    |   │   ├── vllm
+    |   │   ├── wav2lip
+    |   │   └── whisper
     ├── tests
     │   └── embeddings
     │       ├── test_embeddings_tei_langchain.sh
     │       ├── test_embeddings_tei_langchain_on_amd_gpu.sh
     │       └── test_embeddings_tei_llama_index.sh
     └── README.md
-
     ```
 
     - **File Descriptions**:
-      - `embedding_tei.py`: This file defines and registers the microservice. It serves as the entrypoint of the Docker container. Refer to [whisper ASR](https://github.com/opea-project/GenAIComps/tree/main/comps/asr/whisper/README.md) for a simple example or [TGI](https://github.com/opea-project/GenAIComps/blob/main/comps/llms/text-generation/tgi/llm.py) for a more complex example that required adapting to the OpenAI API.
-      - `requirements.txt`: This file is used by Docker to install the necessary dependencies.
-      - `Dockerfile`: Used to generate the service container image. Please follow naming conventions:
-        - Dockerfile: `Dockerfile.[vendor]_[hardware]`, vendor and hardware in lower case (i,e Dockerfile.amd_gpu)
-        - Docker Image: `opea/[microservice type]-[microservice sub type]-[library name]-[vendor]-[hardware]:latest` all lower case (i,e opea/llm-vllm-intel-hpu, opea/llm-faqgen-tgi-intel-hpu-svc)
-
-      - `tests/[microservices type]/` : contains end-to-end test for microservices Please refer to an example [test_asr_whisper.sh](https://github.com/opea-project/GenAIComps/blob/main/tests/asr/test_asr_whisper.sh). Please follow naming convention:`test_[microservice type]_[microservice sub type]_[library name]_on_[vendor]_[hardware].sh`
+      - `deployment/docker_compose` and `deployment/kubernetes`: These folders contain the deployments for the different integrations (i,e for dataprep component we may have compose_pinecone.yaml, compose_milvus.yaml)
```

Comment:
The goal for docker_compose yaml is combine multiple integration into one compose.yaml, so let's remove `(i,e for dataprep component we may have compose_pinecone.yaml, compose_milvus.yaml)`. `dataprep` yaml will be optimized into one file soon.